en/releases/centreon-core.md

```diff
@@ -7,100 +7,108 @@ title: Centreon Core
+  reload the configuration of all pollers from any page
+
+#### Breaking changes
+
+> Access to API v2 has been changed. All of the beta endpoints have been migrated to version 21.10. This must be
+> modified by "latest" or by the version of your Centreon platform (v21.10 for example).
+
+For example replace:
+```shell
+{protocol}://{server}:{port}/centreon/api/beta/login
+```
 
-#### Security
+By:
+```shell
+{protocol}://{server}:{port}/centreon/api/latest/login
+```
 
-- Add SELinux packages
+or:
+By:
+```shell
+{protocol}://{server}:{port}/centreon/api/v21.10/login
+```
 
 #### Performances
 
-- Move to PHP 7.3
-- Move to MariaDB 10.5
+- Move to PHP 8.0
+- Preparing Debian 11 support
 
 ## Centreon Engine
 
-### 21.04.0
+### 21.10.0
 
-- When used with centreon-connectors, Engine could crash when we stop it. This should be fixed now.
-- Ability to submit external commands via gRPC.
-
-> **Warning:** External commands via gRPC are proposed in beta version. The API may change in a near
-future.
+- Flapping now starts only on non-OK states. Based on PR [#523](https://github.com/centreon/centreon-engine/pull/523)
+- Flapping now starts only for services of UP hosts or for hosts with UP parent. Based on PR [#524](https://github.com/centreon/centreon-engine/pull/524). Fixes Issue [#192](https://github.com/centreon/centreon-engine/issues/192)
 
 ## Centreon Broker
 
-### 21.04.0
-
-> **Known issues**
-> * Broker streams: the same parameter, if used in several outputs of the same broker stream, can only have one value (the last prevails).
-> * BAM: the impacts of KPIs of type Meta-service are not evaluated correctly. A fix will be release very soon.
-> * BAM: the impacts of KPIs of type BA in are not evaluated correctly. A fix will be release very soon.
-
-#### New broker logs
-
-- New logs, with new format, epoch timestamps are replaced by real dates.
-
-> **Warning:** you may still see timestamps in your logs until you disable the
-old logs system.
-
-```log
-[2021-04-16 13:49:06.781] [core] [info] Clearing old connections
-[2021-04-16 13:56:10.985] [core] [info] main: configuration update requested
-```
-
-- New log config options, with a different log level for `core`, `config`, `sql`, `processing`, `perfdata`, `bbdo`, `tcp`, `tls`, `lua`, `bam`.
-- Old logs are still supported, but you are encouraged to abandon them.
-
-#### Other enhancements
-
-- Support of UInt64 for `id` column of `index_data` table: fixes issues on platforms having a large amount of metrics.
-
-> **Warning:** this change needs cbd service(s) to be stopped during the upgrade to 21.04.0 and all "queue" and "unprocessed" files must be removed.
+### 21.10.0
 
-- Improvement of the acknowledgement of events when broker is shutting down.
+- Queue (in memory and retention files) are now cleared when the connection is reset. This is a change of behavior back to what it should have always been. It will prevent endless retention files for Centreon-Studio (Centreon-Map).
```

Comment:
```suggestion
- Queues (in memory and retention files) are now cleared when the connection is reset. This is a change of behavior back to what it should have always been. It will prevent endless retention files for Centreon-Studio (Centreon-Map).
```